test/extensions/common/wasm/wasm_vm_test.cc

```diff
@@ -49,7 +49,17 @@ class BaseVmTest : public testing::Test {
   Stats::ScopeSharedPtr scope_;
 };
 
-TEST_F(BaseVmTest, NoRuntime) { EXPECT_EQ(createWasmVm(""), nullptr); }
+TEST_F(BaseVmTest, UnspecifiedRuntime) {
+  auto wasm_vm = createWasmVm("");
+  // Envoy is built with "--define wasm=disabled", so no Wasm engine is available
+  if (getFirstAvailableWasmEngineName().empty()) {
```

Comment:
nit: I'd capture getFirstAvailableWasmEngineName() in a string_view temp rather than calling it twice.

But I'm not hung up over it.